specification/workloads/resource-manager/Microsoft.Workloads/connectors/preview/2023-10-01-preview/connectors.json

```diff
@@ -0,0 +1,1098 @@
+      "x-ms-discriminator-value": "New",
+      "description": "New recovery services vault.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RecoveryServicesVaultProperties"
+        }
+      ],
+      "required": [
+        "name",
+        "resourceGroup"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the recovery services vault has to be created."
+        },
+        "resourceGroup": {
+          "type": "string",
+          "description": "The name of the resource group where the recovery services vault has to be created."
+        }
+      }
+    },
+    "ExistingRecoveryServicesVault": {
+      "x-ms-discriminator-value": "Existing",
+      "description": "Existing recovery services vault.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RecoveryServicesVaultProperties"
+        }
+      ],
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The resource ID of the recovery services vault that has been created."
+        }
+      }
+    },
+    "RecoveryServicesVaultProperties": {
+      "description": "The properties of the recovery services vault used for backup.",
+      "type": "object",
+      "required": [
+        "vaultType"
+      ],
+      "properties": {
+        "vaultType": {
+          "description": "The vault type, whether it is existing or has to be created.",
+          "type": "string",
+          "enum": [
+            "Existing",
+            "New"
+          ],
+          "x-ms-enum": {
+            "name": "vaultType",
+            "modelAsString": true
+          }
+        }
+      },
+      "discriminator": "vaultType"
+    },
+    "VMBackupPolicyProperties": {
+      "description": "Defines the policy properties for virtual machine backup.",
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the VM Backup policy."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "./backupTypes.json#/definitions/AzureIaaSVMProtectionPolicy"
+        }
+      }
+    },
+    "DiskExclusionProperties": {
+      "description": "Defines the disk exclusion properties for virtual machine backup.",
+      "type": "object",
+      "required": [
+        "diskLunList",
+        "isInclusionList"
+      ],
+      "properties": {
+        "diskLunList": {
+          "description": "List of Disks' Logical Unit Numbers (LUN) to be used for VM Protection.",
+          "type": "array",
+          "items": {
+            "format": "int32",
+            "type": "integer"
+          }
+        },
+        "isInclusionList": {
+          "description": "Flag to indicate whether DiskLunList is to be included/ excluded from backup.",
```

Comment:
Would the name includeInBackup be more apt?

Reply:
Similar to the other comment this definition is replicated from Azure backup service swagger https://github.com/Azure/azure-rest-api-specs/blob/5797d78f04cd8ca773be82d2c99a3294009b3f0a/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-04-01/bms.json#L6532.
Keeping the same definitions as of their swagger would help prevent doing service changes to handle property renames when we reference their swagger directly